packages/components/codemods/runV1Codemods/runV1Codemods.ts

```diff
@@ -0,0 +1,187 @@
+    runner: (dir) => {
+      transformComponentsInDir(dir, ['Card'], (tagNames) => [transformCardVariantToColor(tagNames)])
+    },
+  },
+  {
+    name: 'migrateConfirmationModalMoodsToVariant',
+    runner: (dir) => {
+      transformComponentsInDir(dir, ['ConfirmationModal'], (tagNames) => [
+        transformConfirmationModalMoodsToVariant(tagNames),
+      ])
+    },
+  },
+  {
+    name: 'migrateEmptyStateIllustrationTypeToVariant',
+    runner: (dir) => {
+      transformComponentsInDir(dir, ['EmptyState'], (tagNames) => [
+        transformEmptyStateIllustrationTypeToVariant(tagNames),
+      ])
+    },
+  },
+  {
+    name: 'migrateGlobalNotificationTypeToVariant',
+    runner: (dir) => {
+      transformComponentsInDir(dir, ['GlobalNotification'], (tagNames) => [
+        transformNotificationTypeToVariant(tagNames),
+      ])
+    },
+  },
+  {
+    name: 'migrateInformationTileMoodToVariant',
+    runner: (dir) => {
+      transformComponentsInDir(dir, ['InformationTile'], (tagNames) => [
+        transformInformationTileMoodToVariant(tagNames),
+      ])
+    },
+  },
+  {
+    name: 'migrateInlineNotificationTypeToVariant',
+    runner: (dir) => {
+      transformComponentsInDir(dir, ['InlineNotification'], (tagNames) => [
+        transformNotificationTypeToVariant(tagNames),
+      ])
+    },
+  },
+  {
+    name: 'migrateMultiActionTileMoodToVariant',
+    runner: (dir) => {
+      transformComponentsInDir(dir, ['MultiActionTile'], (tagNames) => [
+        transformMultiActionTileMoodToVariant(tagNames),
+      ])
+    },
+  },
+  {
+    name: 'migrateNotificationTypeToVariant',
+    runner: (dir) => {
+      transformComponentsInDir(dir, ['Notification'], (tagNames) => [
+        transformNotificationTypeToVariant(tagNames),
+      ])
+    },
+  },
+  {
+    name: 'migrateProgressBarMoodToColor',
+    runner: (dir) => {
+      transformComponentsInDir(dir, ['ProgressBar'], (tagNames) => [
+        transformProgressBarMoodToColor(tagNames),
+      ])
+    },
+  },
+  {
+    name: 'migrateToastNotificationTypeToVariant',
+    runner: (dir) => {
+      transformComponentsInDir(dir, ['ToastNotification'], (tagNames) => [
+        transformNotificationTypeToVariant(tagNames),
+      ])
+    },
+  },
+  {
+    name: 'migrateWellVariantToColor',
+    runner: (dir) => {
+      transformComponentsInDir(dir, ['Well'], (tagNames) => [transformWellVariantToColor(tagNames)])
+    },
+  },
+  {
+    name: 'removeInputEditModalMood',
+    runner: (dir) => {
+      transformComponentsInDir(dir, ['InputEditModal'], (tagNames) => [
+        removeInputEditModalMood(tagNames),
+      ])
+    },
+  },
+  {
+    name: 'removePopoverVariant',
+    runner: (dir) => {
+      transformComponentsInDir(dir, ['Popover'], (tagNames) => [removePopoverVariant(tagNames)])
+    },
+  },
+]
+
+export const runV1Codemods = async (targetDir: string): Promise<void> => {
+  console.log(`📝 Running ${codemods.length} codemods on directory: ${targetDir}`)
```

Comment:
Should we check if the target directory exists?

Reply:
Good idea - I've added a check and early exit if it fails to find a file